src/lang.js

```diff
@@ -63,7 +63,7 @@
             return key;
         }
 
-        var message = this._getMessage(key);
+        var message = this._getMessage(key, replacements);
```

Comment:
@fgrassals the parameter `replacements` is unused here, can you remove and push again?